covid_ar6_pooled/0_ar6_pooled.py

```diff
@@ -0,0 +1,86 @@
+import click
+import datetime
+from pathlib import Path
+from types import SimpleNamespace
+
+from idmodels.sarix import SARIXModel
+
+@click.command()
+@click.option(
+    "--reference_date",
+    type=str,
+    required=True,
+    help="Reference date for model run (YYYY-MM-DD)",
+)
+@click.option(
+    "--short_run",
+    is_flag=True,
+    help="Perform a short run."
+)
+def main(reference_date: str, short_run: bool):
+    """Get clade counts and save to S3 bucket."""
```

Comment:
I believe this doc line was copied from cladetime's main(). Let's replace it as well as other main() docstrings in this repo ( https://github.com/search?q=repo%3Areichlab%2Foperational-models%20Get%20clade%20counts%20and%20save%20to%20S3%20bucket&type=code ).

Reply:
Fixed in 67d0ee6